scripts/generate-descriptions-md-template

```diff
@@ -25,20 +25,18 @@ tops = list()
 seconds = list()
 
 def truncate(node, cuts, first):
-	if not first and is_cut(node, cuts) and 'ideas' in node:
+	if not first and is_cut(node, cuts) and not len(get_node_children(node)) == 0:
 		reference_title = get_node_reference_title(node)
 		node.update({ 'title': reference_title })
-		if not 'attr' in node:
-			node.update({'attr': dict()})
-		node['attr'].update({'collapsed': True})
+		set_collapsed(node)
 
 	for key, value in node.get('ideas', dict()).iteritems():
 		truncate(value, cuts, False)
 
 	return node
 
 def get_safe_filename(notsafe):
-	safe = notsafe.replace("?", "_").replace("/", "_").replace(",", "_").replace('\'','_').replace('(','_').replace(')','_').replace('  ', ' ')
+	safe = notsafe.replace("?", "_").replace("/", "_").replace(",", "_").replace('\'','_').replace('"','_').replace('(','_').replace(')','_').replace('  ', ' ')
```

Comment:
This is getting unwieldy. Maybe prefer something like:
``safe = re.sub("[?/,'\"()\t]", '_', notsafe)``